### ISSUE TITLE
Change colors and icons

### ISSUE THREAD

**Issue by ifoche** [EyeSeeTea/pictureapp#540]
Finally colors have been accepted and we need to change the renderisation for many screens. Hexadecimal colors are as follow:

Grey: R=135, G=135, B=135 - Hex Triplet: 878786

Orange: R=237, G=156, B=0 - Hex Triplet: ed9c00

Dark purple: R=77, G=59, B=76 - Hex Triplet: 4d3a4b

Purple: R=121, G=93, B=119 - Hex Triplet: 785c77

Light purple: R=156, G=128, B=156 - Hex Triplet: 9c7f9b

Very light purple: R=201, G=184, B=200 - Hex Triplet: c8b8c7

**Comment by idelcano** [comment]
@ifoche i updated the branch, but i will work more tomorrow:

I need do:

- [ ] test the app en some devices..
- [ ] Fix snackbar padding (only looks good in some devices). 



 I did again the next/back buttons and background logic because the other has some problems.
I created a example to set a button text in next/back buttons [commit](https://github.com/EyeSeeTea/pictureapp/pull/567/commits/2d73f4c60e05a5900e832e26faa2bae46b6a3c76)
(it was removed from this branch)

The images and texts are showing as expected for the radiobutton component now( but i will test in more devices tomorrow).


In the monitoring: I can not make the separator line between the left td table rows, the border line always disappears.
